packages/glew.rb

```diff
@@ -3,38 +3,41 @@
 class Glew < Package
   description 'GLEW provides efficient run-time mechanisms for determining which OpenGL extensions are supported on the target platform'
   homepage 'http://glew.sourceforge.net/'
-  version '2.1.0-1'
+  version '2.2.0-rc3'
   compatibility 'all'
-  source_url 'https://github.com/nigels-com/glew/releases/download/glew-2.1.0/glew-2.1.0.tgz'
-  source_sha256 '04de91e7e6763039bc11940095cd9c7f880baba82196a7765f727ac05a993c95'
+  source_url 'https://sourceforge.net/projects/glew/files/glew/snapshots/glew-20200115.tgz/download'
```

Comment:
Why the switch away from github to sourceforge?

Reply:
Github doesn't yet have the ready for use package yet for 2.2.0-rc3, but it is linked to from the github page.
See https://github.com/nigels-com/glew#user-content-downloads:~:text=Recent%20snapshots,-Snapshots